### PR TITLE
update upgrade Notes

### PR DESCRIPTION
**Pull request by putzflorian**
some helpers and informations about moving files and replace strings

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md [hunk 1]
```diff
@@ -47,6 +47,19 @@ Important changes:
 - Move templates from `app/Resources/views` to `templates` and bundles templates from `app/Resources/XYZBundle/views` to `templates/bundles/XYZBundle`
 - Move `app/AppKernel.php` to `src/Kernel.php` and edit it to have `namespace App;` and class name `Kernel`
 - Rename `web/` to `public/` & `web/app.php` to `public/index.php`
+- Rename `config/routing.yml` to `config/routes.yaml`
+- Move your Environment specific configs to `config/packages/$ENV/'.yaml`
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md]: Something is not right here... (`'` as filename?)

Comment (putzflorian) [reply doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md]: sorry there was a typo issue.
thx for the info 👍

**Review by putzflorian — COMMENTED** [review]
typo

File: doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md [hunk 2]
```diff
@@ -56,6 +69,76 @@ Since Pimcore X supports only Symfony 5, which dropped the support for Php templ
 
 If you are using enterprise edition, then it is still possible to support Php templates by installing `pimcore/php-templating-engine-bundle`.
 
+### Additional Changes for Templates
```
Comment (dvesh3) [inline doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md]: I would keep this as a sub section of "Migrate Php templates to Twig" and maybe move RegEx Examples to gist

Comment (putzflorian) [reply doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md]: External Link is implemented and is moved to the "Migrate Php templates to Twig" Section

**Review by putzflorian — COMMENTED** [review]
External RegEx Link is implemented

**Comment by dvesh3** [comment]
thanks a lot!

**Pull request merged**